keyboards/zlabkeeb/15pad/readme.md

```diff
@@ -0,0 +1,25 @@
+# 15PAD
+
+![15PAD](https://i.imgur.com/J7sZSnxh.jpeg)
```

Comment:
This requirement has been changed, as it seems no longer supported by Imgur.
```suggestion
![15PAD](https://i.imgur.com/J7sZSnx.jpeg)
```

Reply:
okay thanks